dist/vee-validate.esm.js

```diff
@@ -741,13 +778,24 @@ ErrorBag.prototype._selector = function _selector (field) {
 /**
  * Get the field scope if specified using dot notation.
  */
-ErrorBag.prototype._scope = function _scope (field) {
-  if (field.indexOf('.') > -1) {
-    var ref = field.split('.');
-      var scope = ref[0];
-      var name = ref.slice(1);
+ErrorBag.prototype._scope = function _scope (field, context) {
+  while (context) {
+    if (context.$parent.$vnode.componentOptions.tag === 'WForm') {
```

Comment:
@gauravmuk we can get the form tagname using this.
$vnode.componentInstance.$el.tagName

And the data-vv-instance by 
$vnode.componentInstance.$el.getAttribute('data-vv-scope')